frontend/src/components/dashboard/widgets/TaskList.tsx

```
@@ -203,161 +229,232 @@ export const TaskList = ({
-                    newLevel: taskAbove.level + 1,
+                    newParentId: targetTask.parentId
+                        ? parseInt(targetTask.parentId)
+                        : undefined,
+                    newDisplayOrder: targetTask.displayOrder,
+                };
+            case 'child':
+                // Become child of target task
+                // Find the highest display order among target's children and add 1
+                const targetChildren = flatTasks.filter(
+                    (t) => t.parentId === targetTask.id,
+                );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)